src/util/tests/envVariablesValidations.test.ts

```diff
@@ -75,8 +75,4 @@ describe('Validate .env file', () => {
   test('expected ENV variables are set', () => {
     expect(checkExpectedVariables()).toBeTruthy();
   });
-
```

Comment:
Needed to remove this since now some variables are optional.

Reply:
We can instead list the optional ones?